interfaces/builtin/greengrass_support.go

```diff
@@ -392,12 +392,19 @@ mknod - |S_IFCHR -
 mknodat - - |S_IFCHR -
 `
 
+// XXX: decide on the final names and adjust tests too
```

Comment:
> This commit adds the TODO 

so TODO or XXX?

Reply:
Sorry, TODO

Reply:
I'm just using this PR to fix up the actual name as discussed, so this TODO goes awa